src/components/DeveloperUILinkButton/DeveloperUILinkButton.tsx

```diff
@@ -7,16 +8,30 @@ import './DeveloperUILinkButton.scss';
 
 const b = cn('developer-ui-link-button');
 
+const buttonSizeToIconSize: Record<ButtonSize, number> = {
+    xs: 14,
+    s: 16,
+    m: 16,
+    l: 18,
+    xl: 18,
+};
+
 interface DeveloperUiLinkProps {
     className?: string;
     visible?: boolean;
     href: string;
+    size?: ButtonSize;
 }
 
-export function DeveloperUILinkButton({href, visible = false, className}: DeveloperUiLinkProps) {
+export function DeveloperUILinkButton({
+    href,
+    visible = false,
+    className,
+    size = 's',
+}: DeveloperUiLinkProps) {
     return (
-        <Button size="s" href={href} className={b({visible}, className)} target="_blank">
-            <Icon data={ArrowUpRightFromSquare} />
+        <Button size={size} href={href} className={b({visible}, className)} target="_blank">
+            <Icon data={ArrowUpRightFromSquare} size={buttonSizeToIconSize[size]} />
```

Comment:
I suppose we can use Button Icons as here

```
       <Button
                view="flat"
                onClick={() => {
                    dispatch(api.util.invalidateTags(['All']));
                }}
                extraProps={{'aria-label': i18n('Refresh')}}
            >
                <Button.Icon>
                    <ArrowsRotateLeft />
                </Button.Icon>
            </Button>
 ```
 
 
 and icon size should be set automatically according to button size

Reply:
Discussed with uikit team, button doesn't affect it's icon size. We should explicitly set icon size we want.